src/components/@shared/AssetTeaser/index.tsx

```diff
@@ -51,13 +53,15 @@ export default function AssetTeaser({
             <Dotdotdot tagName="h1" clamp={3} className={styles.title}>
               {name.slice(0, 200)}
             </Dotdotdot>
-            {!noPublisher && <Publisher account={owner} minimal />}
+            {!noPublisher && !noDescription && <Publisher account={owner} />}
```

Comment:
Adding `noDescription` here is confusing and redundant. There is already a prop to hide the publisher, so use this by adding the `noPublisher` prop to the component

Reply:
Good point, I've removed it